charts/pocket-validator/README.md

```diff
@@ -0,0 +1,130 @@
+| image.tag | string | `"latest"` | image tag |
+| imagePullSecrets | list | `[]` | image pull secrets |
+| ingress.annotations | object | `{}` |  |
+| ingress.className | string | `""` |  |
+| ingress.enabled | bool | `false` | enable ingress for RPC port |
+| ingress.hosts[0].host | string | `"chart-example.local"` |  |
+| ingress.hosts[0].paths[0].path | string | `"/"` |  |
+| ingress.hosts[0].paths[0].pathType | string | `"ImplementationSpecific"` |  |
+| ingress.tls | list | `[]` |  |
+| nameOverride | string | `""` |  |
+| nodeSelector | object | `{}` |  |
+| persistence.accessModes | list | `["ReadWriteOnce"]` | persistent Volume Access Modes |
+| persistence.annotations | object | `{}` | annotations of the persistent volume claim |
+| persistence.dataSource | object | `{}` | custom data source of the persistent volume claim |
+| persistence.enabled | bool | `true` | enable persistent volume claim |
+| persistence.existingClaim | string | `""` | name of an existing PVC to use for persistence |
+| persistence.reclaimPolicy | string | `"Delete"` | persistent volume reclaim policy |
+| persistence.selector | object | `{}` | selector to match an existing Persistent Volume |
+| persistence.size | string | `"8Gi"` | size of the persistent volume claim |
+| persistence.storageClass | string | `""` | storage class of the persistent volume claim |
+| podAnnotations | object | `{}` | pod annotations |
+| podSecurityContext | object | `{}` |  |
+| postgresql.enabled | bool | `true` | deploy postgresql database automatically. Refer to https://github.com/bitnami/charts/blob/main/bitnami/postgresql/values.yaml for additional options. |
+| postgresql.primary.persistence.enabled | bool | `false` | enable persistent volume claim for PostgreSQL |
+| postgresql.primary.persistence.size | string | `"8Gi"` | size of the persistent volume claim for PostgreSQL |
+| privateKeySecretKeyRef.key | string | `""` | REQUIRED. Key in the Secret that contains the private key of the node |
+| privateKeySecretKeyRef.name | string | `""` | REQUIRED. Name of the Secret in the same namespace that contains the private key of the node |
+| resources | object | `{}` | resources limits and requests |
+| securityContext | object | `{}` |  |
+| service.annotations | object | `{}` | service annotations |
+| service.ports.consensus | int | `42069` | consensus port of the node |
+| service.ports.metrics | int | `9000` | OpenTelemetry metrics port of the node |
+| service.ports.rpc | int | `50832` | rpc port of the node |
+| service.type | string | `"ClusterIP"` | service type |
+| serviceAccount.annotations | object | `{}` | Annotations to add to the service account |
+| serviceAccount.create | bool | `true` | Specifies whether a service account should be created |
+| serviceAccount.name | string | `""` | The name of the service account to use. If not set and create is true, a name is generated using the fullname template |
+| tolerations | list | `[]` |  |
```

Comment:
Should we add a GITHUB_WIKI tag here?

Reply:
@Olshansk the vast majority of open-source helm charts have that kind of information (values, examples, etc.) in the README.md in the root of the chart. I think that is the place where users would look. To be frank, I don't think we need this in the wiki.